NewSpring/ApollosProject
# [DEV-3881] Refactor Add Prayer Form

## DESCRIPTION

### What does this PR do, or why is it needed?

Part of the Rosary Refactor:

This updates the Add Prayer Form to align with the new design. 

Minor tweaks include:
- Full-screen modal. No app menu items should be visible.
- Left-align all the things.
- Add a rounded border around the prayer text area.
- “Share Anonymously” should be immediately after the prayer text area.
- Round the “Share prayer” button.

NOTE: The AvoidKeyBoardView makes this funky. Normally we would wrap `PaddedView` with a `FlexedView` for things like this, but because of the Avoid, I'm giving the `FlexedView`s padding.

### How do I test this PR?

Open up a sim and make sure that the above are correct. Run tests.

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO


## REVIEW


**Manual QA**


---

> The purpose of PR Review is to _improve the quality of the software._
